src/XrdHttp/XrdHttpProtocol.cc

```diff
@@ -1778,6 +1661,14 @@ int XrdHttpProtocol::Configure(char *parms, XrdProtocol_Config * pi) {
 /******************************************************************************/
 
 bool XrdHttpProtocol::InitSecurity() {
```

Comment:
Moving this should be pretty trivial so I don't understand why it seemed difficult. I guess because the sprinkling of variables is rather baroque. The key is understanding what InitSecurity() relies on (and it's not much):
a) You need to include XrdHttpTace.hh
b) You need to add
namespace XrdHttpProtoInfo
{
extern XrdTlsContext *xrdctx;
}
using namespace XrdHttpProtoInfo;
c) You need to include XrdTlsContext.hh
d) You need to include XrdOucGMap.hh which can then be deleted from XrdHttpProtocol.cc itself

That's pretty much it! So please try again.

Reply:
Move in the new PR.